README.md

```diff
@@ -39,8 +39,10 @@ GF_INSTALL_PLUGINS=grafana-pyroscope-app
 2. You’ll land in the service overview page that shows time series and CPU utilization visualizations for all the services in your selected Pyroscope instance.
 3. Change your data source with the drop-down on the top left.
 4. Modify your time range in two ways:
-  * Use the standard time range picker on the top right.
-  * Click and drag the time range you want to see on any time series visualization.
+
+- With the standard time range picker on the top right.
+- By clicking and dragging the time range you want to see on any time series visualization.
+
```

Comment:
The parser has misinterpreted this. It should be an unordered list nested in step 4.:

```suggestion

   - With the standard time range picker on the top right.
   - By clicking and dragging the time range you want to see on any time series visualization.

```